### ISSUE TITLE
Feeds Plugin: Link to Web Resource

### ISSUE THREAD

**Issue by gvautour** [wet-boew/wet-boew#8571]
Should we use the atom:link to link to a web resource instead of the atom:id? I believe that the use of the id is a deviation from previous implementations of the feeds plugin, but I could be mistaken.

According to the RFC4287, the [atom:link](https://tools.ietf.org/html/rfc4287#section-4.2.7) defines a reference from an entry or feed to a Web resource while the [atom:id](https://tools.ietf.org/html/rfc4287#section-4.2.6) conveys a permanent, universally unique identifier for an entry or feed.

**Comment by duboisp** [comment]
I think the wb-feeds use the first atomLink not the atomId as I can see at line: https://github.com/wet-boew/wet-boew/blob/master/src/plugins/feeds/feeds.js#L99 and https://github.com/wet-boew/wet-boew/blob/master/src/plugins/feeds/feeds.js#L324

**Comment by gvautour** [comment]
That is correct, however, while it it being processed in the corsEntry function, the id is assigned to the link. https://github.com/wet-boew/wet-boew/blob/master/src/plugins/feeds/feeds.js#L249

**Comment by duboisp** [comment]
@ipaksc can you take a look at this? Can we try to use the "link" first then the "id" if unavailable?
